test/e2e/page-objects/pages/dialog/notification.ts

```diff
@@ -0,0 +1,19 @@
+import { Driver } from '../../../webdriver/driver';
+
+class Notification {
+  private driver: Driver;
+
+  private submitButton: string;
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+    this.submitButton = '[data-testid="confirmation-submit-button"]';
+  }
+
+  async clickApproveButton(): Promise<void> {
+    console.log('Click Approve Button');
+    await this.driver.clickElement(this.submitButton);
```

Comment:
to avoid race conditions:

```suggestion
    await this.driver.clickElementAndWaitToDisappear(
      this.submitButton,
    );
```